src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -145,17 +181,21 @@ public bool HasNoResults
             get { return this.SearchResults.Count == 0; }
         }
 
+
+        public PackageSearchState _searchState;
```

Comment:
@RyaPorter Can you add a comment suggesting that this backing field be set to private in Dynamo 3.0?  Doing so now would unfortunately introduce an API breaking change.